gnome-initial-setup/gis-driver.c

```diff
@@ -862,6 +862,31 @@ window_realize_cb (GtkWidget *widget, gpointer user_data)
   update_screen_size (GIS_DRIVER (user_data));
 }
 
+static void
+emit_startup_sound (GApplication * app, gpointer user_data)
+{
+  GDBusProxy *proxy;
+  GError *error = NULL;
+
+  proxy = g_dbus_proxy_new_for_bus_sync (G_BUS_TYPE_SESSION,
+                                         G_DBUS_PROXY_FLAGS_NONE,
+                                         NULL,
+                                         "com.endlessm.HackSoundServer",
+                                         "/com/endlessm/HackSoundServer",
+                                         "com.endlessm.HackSoundServer",
+                                         NULL, &error);
+  if (error) {
+    g_printerr ("Cannot stablish proxy to 'com.endlessm.HackSoundServer': %s",
```

Comment:
"establish", but I think a better message would be "Cannot create proxy for '...'"